core/dbt/task/runnable.py

```diff
@@ -206,17 +206,34 @@ def call_runner(self, runner):
         with RUNNING_STATE, uid_context:
             startctx = TimestampNamed('node_started_at')
             index = self.index_offset(runner.node_index)
+            runner.node.config['started_at'] = datetime.utcnow().isoformat()
```

Comment:
Is there any possibility that these could interfere with user-provided configs for `started_at` + `finished_at`? If so, could we name these in such a way that collisions are extremely unlikely (e.g. `dbt_internal__started_at`)?